src/game_objects/game_object.h

```diff
@@ -24,13 +29,16 @@ class GameObject {
     btRigidBody *rigidbody;
     int id;
 public:
+    static std::unordered_map<int, GameObject*> game_objects;
     std::vector<GameObject *> children;
 
     Transform transform; // World matrix now controlled using the Transform Component
-    std::string tag; // Identify this GameObject by a human-readable tag.
+    glm::vec3 direction;
+    Tag tag; // Identify this GameObject by a tag.
```

Comment:
Ah, it didn't seem like we were using this tag, so I repurposed it.  I can change it back and switch to dynamic casts.